docs/src/lessons/XYPlots/examples/BarChartWithDefs.js.example

```diff
@@ -0,0 +1,27 @@
+const BarChartWithDefs = (props) => {
+  const data = [
+    {x: 0, y: 80},
+    {x: 5, y: 60},
+    {x: 10, y: 90},
+    {x: 15, y: 30},
+  ];
+  return <XYPlot width={400} height={300} defs={
+    <linearGradient id="Gradient" x1="0" x2="0" y1="0" y2="1">
+      <stop offset="0%" stopColor="blue" />
+      <stop offset="50%" stopColor="white" />
+      <stop offset="100%" stopColor="red" />
+    </linearGradient>
+  }>
+    <XAxis showGrid={false} title="Days since Zombiepocalypse" />
```

Comment:
@scottsheffield Try changing this so instead of passing in defs, define defs as the first child. For some reason React doesn't like it and the chart doesn't render. Any idea why?

```js
  <defs>
    <linearGradient id="Gradient" x1="0" x2="0" y1="0" y2="1">
      <stop offset="0%" stopColor="blue" />
      <stop offset="50%" stopColor="white" />
      <stop offset="100%" stopColor="red" />
    </linearGradient>
  </defs>
```